style.css

```diff
@@ -175,13 +184,254 @@ body {
 
-@media (max-width: 1235px) {
+.second {
+  background-color: #FAE8E4;
+  height: 900px;
+  width: 100%;
+}
+
+.second-container {
+  display: grid;
+  grid-template-columns: 730px 488px;
+  grid-template-rows: 593px; /*622*/
+  grid-column-gap: 42px;
+  padding-top: 172px;
+  justify-content: center;
+}
+
+.second-container__text-white {
+  width: 488px;
+  height: 556px;
+  background-color: #fff;
+  box-shadow: 8px 8px 25px rgba(122,122,122,0.5);
+}
+
+.second-container__text-white-container {
+  width: 378px;
+  height: 556px;
+  margin: 0 auto;
+  margin-bottom: 34px;
+}
+
+.second-container__text-title {
+  font-size: 40px;
+  line-height: 1;
+  margin: 0;
+  padding-top: 40px;
+}
+
+.second-container__bags {
+  padding-top: 60px;
+}
+
+.second-container__text-subtitle {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 1.1;
+  letter-spacing: .07em;
+  opacity: .5;
+}
+
+.second-container__text-main {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 1.4;
+  opacity: .5;
+  margin-top: 65px;
+}
+
+.second-container__text-color {
+  display: flex;
+  flex-direction: row;
+}
+
+.second-container__text-gray,
+.second-container__text-pink,
+.second-container__text-blue,
+.second-container__text-green {
+  width: 30.18px;
+  height: 30px;
+  border-radius: 50%;
+}
+
+.second-container__text-gray:hover,
+.second-container__text-pink:hover,
+.second-container__text-blue:hover,
+.second-container__text-green:hover {
+  transform: scale(1.2);
+  transition: 1s ease-out .2s;
+  cursor: pointer;
+}
+
+.second-container__text-gray {
+  background-color: #DEDEDE;
+}
+
+.second-container__text-pink {
+  background-color: #F29F9F;
+  margin-left: 11.82px;
+}
+
+.second-container__text-blue {
+  background-color: #A8BFF9;
+  margin-left: 11.82px;
+}
+
+.second-container__text-green {
+  background-color: #90BD8F;
+  margin-left: 11.82px;
+}
+
+.second-container__text-coust {
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 1.3;
+  letter-spacing: 2px;
+  margin: 0;
+}
+
+.second-container__text-block {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 42px;
+  margin-right: 18px;
+  margin-bottom: 59px;
+}
+
+.second-container__text-button-quantity {
+  text-decoration: none;
+  color: #000;
+  font-style: normal;
+  font-size: 15px;
+  border: 1.5px solid #000;
+  padding: 11px 35px;
+  margin-right: 9px;
+}
+
+.second-container__text-button-basket{
+  text-decoration: none;
+  color: #000;
+  font-style: normal;
+  font-size: 13px;
+  border: 1.5px solid #000;
+  padding: 11px 44px;
+}
+
+.second-container__text-other {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 1.2;
+  text-decoration: none;
+  color: #000;
+  margin-left: 5px;
+  margin-right: 10px;
+}
+
+.third {
+  background-color: #EBE3D8;
```

Comment:
Codacy found an issue: [Expected "#EBE3D8" to be "#ebe3d8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)